apps/api/src/database/schema.prisma

```diff
@@ -663,6 +665,37 @@ model ListedBuildingDetails {
   lpaQuestionnaire      LPAQuestionnaire? @relation(fields: [lpaQuestionnaireId], references: [id])
 }
 
+// ProjectUpdate represents some content in relation to a case (application/nsip) that may be of interest to
+// members of the public and those with an interest in that case. These updates are ultimate sent out to email
+// subscribers, and published to the front office.
+// Otherwise known as Banners or Latest Updates.
+model ProjectUpdate {
+  id                Int       @id @default(autoincrement())
+  // the ID of the case this project update relates to
+  caseId            Int
+  case              Case?     @relation(fields: [caseId], references: [id])
+  // the ID of the user who created this update
+  authorId          Int?
+  author            User?     @relation(fields: [authorId], references: [id])
+  // when was this project update first created
+  dateCreated       DateTime  @default(now())
+  // should this update be emailed to subscribers?
+  emailSubscribers  Boolean   @default(false)
+  // status of this update
+  status            String    @default("draft") // draft | published | unpublished | archived
+  // when was this project update changed to status "published"
+  datePublished     DateTime?
+  // when the (background) task to notify subscribers has been run for this update, this will be marked true
+  sentToSubscribers Boolean   @default(false)
+  // an internal title/name for this update
+  title             String?
```

Comment:
Just wanted to confirm title is optional, I don't have full context so I'm guessing it is.

Reply:
Yup, note added: https://github.com/Planning-Inspectorate/back-office/pull/992/commits/896ea08bb5ca4bfdc3dd1fc0970e30531b55c961

Internal title might become a field on the UI, but for MVP won't be there.